Formula/gimp.rb

```diff
@@ -1,87 +0,0 @@
-class Gimp < Formula
```

Comment:
@xu-cheng See the linked issue: https://github.com/caskroom/homebrew-cask/issues/15603